examples/next/pages/ui/components/in-app-messaging/demo/utils.ts

```diff
@@ -1,12 +1,12 @@
 import { useState, useEffect } from 'react';
-import { Notifications } from '@aws-amplify/notifications';
 import * as Analytics from '@aws-amplify/analytics';
+import { syncMessages } from 'aws-amplify/in-app-messaging';
 import {
-  InAppMessage,
-  InAppMessageAction,
-  InAppMessageButton,
-  InAppMessageLayout,
-} from '@aws-amplify/notifications';
+  Message,
```

Comment:
Wasn't sure if this was the right call but changed these to not expose the js types (because we were already reassigning them like type = jstype in the notifications package)

Reply:
Do we export `InAppMessage` from ui-react-notifications? If not let's expose and extend from it to replace these types, example:
```ts
type MessageLayout = Message['layout']
```

Reply:
We don't export it directly, it's like this: 
``` 
export type MessageLayout = InAppMessageLayout; 
```
Do you mean like just import Message into the example and then define the layout types etc in the example?

Reply:
Close - what I'm asking is that we export `InAppMessage` from _@aws-amplify/ui-react-notifications_ (if we do not already) as it is a public package (and the type itself could provide value to consumers), and then create utility types from it as needed in this file

Reply:
But we're already exporting Message, MessageAction etc so should we stop doing that in favor of only exporting InAppMessage? I feel like it would be harder if you have to reconstruct the types every time you want to use them

Reply:
Let's handle this as a follow up 🙂